devtools/cli/src/main/java/io/quarkus/cli/Completion.java

```diff
@@ -4,12 +4,7 @@
 import picocli.CommandLine;
 
 @CommandLine.Command(name = "completion", version = "generate-completion "
-        + CommandLine.VERSION, mixinStandardHelpOptions = true, description = {
-                "bash/zsh completion:  source <(${PARENT-COMMAND-FULL-NAME:-$PARENTCOMMAND} ${COMMAND-NAME})",
-                "Run the following command to give `${ROOT-COMMAND-NAME:-$PARENTCOMMAND}` TAB completion in the current shell:",
-                "",
-                "  source <(${PARENT-COMMAND-FULL-NAME:-$PARENTCOMMAND} ${COMMAND-NAME})",
-                "" }, optionListHeading = "Options:%n", helpCommand = true)
+        + CommandLine.VERSION, mixinStandardHelpOptions = true, header = "bash/zsh completion:  source <(${PARENT-COMMAND-FULL-NAME:-$PARENTCOMMAND} ${COMMAND-NAME})", helpCommand = true, headerHeading = "%n", commandListHeading = "%nCommands:%n", synopsisHeading = "%nUsage: ", optionListHeading = "Options:%n")
```

Comment:
are this introduction of *Heading attributes just to create bit more "air" with extra newlines or ?

Reply:
Yes.